src/include/stir/recon_buildblock/ProjMatrixByBinSPECTUB.h

```diff
@@ -172,6 +171,9 @@ class ProjMatrixByBinSPECTUB :
 
   bool already_setup;
 
+  static wm_da_type wm;
```

Comment:
I think this has to be
```
static SPECTUB::wm_da_type wm;
```

Reply:
removing statis causes the following:
`
ProjMatrixByBinSPECTUB.cxx:879: error: binding reference of type ‘SPECTUB::wm_da_type&’ to ‘const SPECTUB::wm_da_type’ discards qualifiers
ProjMatrixByBinSPECTUB.cxx: In member function ‘virtual void stir::ProjMatrixByBinSPECTUB::calculate_proj_matrix_elems_for_one_bin(stir::ProjMatrixElemsForOneBin&) const’:
ProjMatrixByBinSPECTUB.cxx:879:30: error: binding reference of type ‘SPECTUB::wm_da_type&’ to ‘const SPECTUB::wm_da_type’ discards qualifiers
  879 |       compute_one_subset(kOS,wm,wmh,Rrad);
                                    ^~
ProjMatrixByBinSPECTUB.cxx:741:32: note:   initializing argument 2 of ‘void stir::ProjMatrixByBinSPECTUB::compute_one_subset(int, SPECTUB::wm_da_type&, SPECTUB::wmh_type&, float*) const’
  741 |                    wm_da_type& wm,
      |                    ~~~~~~~~~~~~^~ '

Reply:
hmmm. probably a conflict between the member variable and the argument. It is of course a bit confusing now. I guess either rename arguments and where they are used, or remove arguments and use member variables. Maybe the latter is more logical after all. In that case, it seems `compute_one_subset` can no longer be `const`. If it has to be, then we'd have to make the members `mutable`.